test/stdlib/TestCharacterSet.swift

```diff
@@ -289,8 +289,17 @@ class TestCharacterSet : TestCharacterSetSuper {
             expectNotEqual(abcSet, abcdSet)
             
             expectTrue(abcSet.isStrictSubset(of:abcdSet))
-            expectFalse(abcdSet.isStrictSubset(of:abcSet))
-            
+            #if (arch(i386) || arch(x86_64)) && os(iOS)
+            let vers = ProcessInfo.processInfo.operatingSystemVersion
+            if vers.majorVersion >= 10 && vers.minorVersion >= 3 && vers.patchVersion >= 0 {
+                expectFalse(abcdSet.isStrictSubset(of:abcSet))
+            } else {
+                print("running on \(vers) but was guarded to 10.3")
```

Comment:
Is it possible the simulator CF's NSCharacterSet is implemented by just calling the host?